### DISCUSSION TITLE
[BUG]无法修改php容器sysctl

### DISCUSSION THREAD

**Discussion by gongcf** [1Panel-dev/1Panel#6924] [Q&A]
### 1Panel 版本

v1.10.20-lts

### 问题描述

无法修改php容器sysctl
正常在linux环境可以使用sysctl -w net.ipv4.tcp_tw_reuse=1进行修改，但在容器只报错提示为只读

查看docker-compose相关说明,要配置sysctls段，但管理后台现没有配置的地方。

应如何处理？

### 重现步骤

进入php容器输入：sysctl -w net.ipv4.tcp_tw_reuse=1

### 期待的正确结果

可正常修改参数

**Comment by wanghe-fit2cloud** [comment]
可以尝试下面的方案是否可行，如图：
![image](https://github.com/user-attachments/assets/3cff7e48-466a-446c-8ad8-bc3bce57c53e)
